### DISCUSSION TITLE
Kaspersky Internet Security and Atlas OS

### DISCUSSION THREAD

**Discussion by lAL3Xl** [Atlas-OS/Atlas#1184] [Help & Support] [answered]
I have installed Atlas OS  and for the installation I removed Kaspersky Internet Security from my Laptop.
After the installation of Atlas was completed I installed Kaspersky again. It detected a file named: FILEASSOC.cmd as a HEUR:Trojan.BAT.Agent.gen and removed it. The install path was: C:\Users\User\AppData\Local\Temp\AME\Playbooks\Atlas-AtlasOS\Executables

Idk what the file did but Kaspersky removed it and everything still works (up until now). 

Can someone explain what the file does and if it was bad. Maybe the bug can be fixed because i don´t think that this was a Trojan.

EDIT: I have installed Atlas OS 0.4 (the newest) on Windows 10

**Comment by H0ggu** [comment, marked as answer]
This script, just changes the default app for specific file types, so the browser you select during install is set as default, and so 7zip has the correct file extensions associated with it.
This file is in the temp folder for the AME wizard, since you've already applied the playbook it doesn't matter if this file has been removed
https://github.com/Atlas-OS/Atlas/blob/6fc948332a3c605e489eaa27918d9904cd3986fa/src/playbook/Executables/FILEASSOC.cmd#L4
The above link is to the specific file in reference, it is not malicious
